hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/EnrichmentProcessor.java

```diff
@@ -56,7 +73,7 @@ public void process(StructuredTrace trace) {
     AvroToJsonLogger.log(LOG, "Structured Trace after all the enrichment is: {}", trace);
   }
 
-  private void applyEnricher(Enricher enricher, StructuredTrace trace) {
+  private Void applyEnricher(Enricher enricher, StructuredTrace trace) {
```

Comment:
? why the change to null? I'm guessing because we're trying to provide a callable, but guava takes a runnable version too.

Reply:
We can use runWithTimeout instead